services/graphql-server/src/graphql/resolvers/website/section.js

```diff
@@ -60,6 +60,21 @@ module.exports = {
       return sections.reverse();
     },
 
+    /**
+     * Returns the website sections directly related to this section. This is primarily used for the
+     * Leaders Program to denote a contextual relationship between two website sections.
+     */
+    relatedSectionIds: async ({ relatedSections }) => relatedSections || [],
+
+    /**
+     * Returns the taxonomy terms directly related to this section. This is primarily used for the
+     * Leaders Program to denote a contextual relationship.
+     */
+    relatedTaxonomyIds: async ({ relatedTaxonomy }) => {
+      const refs = relatedTaxonomy || [];
+      return refs.map(BaseDB.extractRefId);
```

Comment:
In this case: 
`relatedTaxonomyIds: (doc) => getAsArray(doc, 'relatedTaxonomy').map(BaseDB.extractRefId),`